config/initializers/launch_darkly.rb

```diff
@@ -1,3 +1,3 @@
 # This is actually handled on production by config/puma.rb in the `on_worker_boot` section.
 # This configuration is just for localhost development.
-$launch_darkly = LaunchDarkly::LDClient.new(ENV.fetch("LAUNCHDARKLY_SDK_KEY")) if Rails.env.development?
+$launch_darkly = LaunchDarkly::LDClient.new(ENV.fetch("LAUNCHDARKLY_SDK_KEY")) if Rails.env.test? || Rails.env.development?
```

Comment:
Maybe it's better just to do `unless Rails.env.production?` and delete the comment on line 2?

Reply:
Actually I'm retarded - I think this needs to happen always, since if we run rake tasks on prod LaunchDarkly won't be loaded correctly.

So the correct thing is to just remove the `if` statement and always load `$launch_darkly` both here in the initializer and in the Puma config.

Reply:
The reason we need to also load it in the Puma config is because Puma is multithreaded so unless the instantiation happens there, sometimes a Puma thread won't get the `$launch_darkly` variable initialized :(

Reply:
Yea okay, makes sense. Maybe worth exploring a better solution at a later stage though so we don't need to duplicate this logic